apps/next/utils/getRemoteAssets.ts

```diff
@@ -0,0 +1,12 @@
+import { getPlaiceholderImage } from 'app/utils/getPlaiceholderImage'
+
+export async function getRemoteAssets(paths: string[] = []) {
+  const remoteImageUrl = 'https://github.com/0xsend/assets/blob/main/'
+  const images = []
+  for (let i = 0; i < paths.length; i++) {
```

Comment:
Load them all at the same time instead of waiting. Should be ok since we won't be doing 100s of images at time but should keep load times under 100-200ms.

```suggestion
  const imagePromises = paths.map(path => getPlaiceholderImage(remoteImageUrl + path))
  const images = await Promise.all(imagePromises)
```